recipes/si/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.8)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
```

```suggestion
conan_basic_setup(TARGET)
```

For header-only recipes, you have to use TARGET, otherwise conan_basic_setup injects the header folder globally and SI::SI will be shadowed, so even either SI::SI be incorrect, the test package won't break.